media/css/firefox/family/components/_hero.scss

```
@@ -39,8 +39,59 @@ $image-path: '/media/protocol/img';
     }
 
     .c-blurb {
-        background-color: f3.$green-light;
+        @include text-body-lg;
+        @include font-firefox;
+
+        --shadow-offset: #{$border-radius-md};
+        @include f3.card-shadow;
+        background-color: f3.$blue-light;
+        border-radius: $border-radius-lg;
+        color: inherit;
+        display: inline-block;
+        font-weight: bold;
         margin-top: $layout-lg;
+        text-align: center;
+        text-decoration: none;
+
+        @media (prefers-reduced-motion: no-preference) {
+            transition: transform 0.2s;
+
+            &:hover,
+            &:focus {
+                transform: scale(1.05);
+            }
+
+            @media #{$mq-sm} {
+                &:hover,
+                &:focus {
+                    transform: scale(1.2);
+                }
+            }
+        }
+
+        @media (prefers-reduced-motion: reduce) {
+            &:hover,
+            &:focus {
+                text-decoration: underline;
+                text-decoration-thickness: 2px;
+            }
+        }
+
+        @media #{$mq-md} {
+            display: flex; // allow logo to share same line as text
+        }
+    }
+
+    .c-blurb-logo {
+        @include background-size(222px, 26px);
+        @include image-replaced;
+        background-image: url('/media/img/firefox/family/logo-common-sense-media.svg');
+        background-repeat: no-repeat;
+        background-position: center;
+        background-size: contain;
+        max-width: 222px;
+        min-width: 20ch;
```

This button is just a little too big at 360px and while there's no horizontal scroll bar everything ends up a bit off centre.
```suggestion
        min-width: 18ch;
```